ldap-testing/config.php

```diff
@@ -1,7 +1,23 @@
 <?php
+  // Basic params
   $bdn = 'dc=owncloud,dc=com';
   $adn = 'cn=admin,dc=owncloud,dc=com';
   $apwd = 'admin';
   $host = 'localhost';
   $port = 389;
 
+  // Define amount of users in groups
+  $amount_users_in_groups = 0;
```

Comment:
Yes, @PVince81 by default they have no members, only zombieKing. Should we make it more transparent? (README)?

Reply:
can you set the default to some non-zero value maybe ?

Reply:
makes sense, otherwise it is just confusing